docs/feature_midi.md

```diff
@@ -0,0 +1,260 @@
+# MIDI
+
+## Enabling MIDI
+
+Add the following to the keyboard's rules.mk
```

Comment:
```suggestion
## Usage

First, enable MIDI by adding the following to your `rules.mk`:
```

Bit of wordsmithing here. Had to break it up because of the triple backticks that are also used in suggestion blocks...